### PR TITLE
Feat: Update ME Logo and Move Email Templates to Postmark

### LINKED ISSUE #686: Convert all emails to use the Postmark templates
**Description**
the routine send_massenergize_rich_email ios used  in many places to send email using the old HTML templates.
These templates should be migrated to PostMark templates.

### LINKED ISSUE #1114: Replace old MassEnergize logo with new one
**Description**
In any email template that includes the old MassEnergize logo - change it to the new one:

https://www.massenergize.org/wp-content/uploads/2023/03/MassEnergize-logo.png

The old logo was at:
https://www.massenergize.org/wp-content/uploads/2021/07/cropped-me-logo-transp.png

There are 8 files in the API repository that reference the old logo.  If possible, define the logo in one place, and refer to it indirectly from the numerous locations.

In addition, update the logo on the PostMark templates for all servers/

### PR DESCRIPTION
**Pull request by abdullai-t**
####  Summary / Highlights
- Updated MassEnergize logo across all email templates using the new `ME_LOGO_PNG` constant
- Migrated email templates to Postmark for improved email handling
- Refactored email template handling for multiple features including user registration, team invitations, admin notifications, and more

#### Details (Give details about what this PR accomplishes, include any screenshots etc)
This PR accomplishes two main objectives:

1. **Logo Update**
- Replaced old MassEnergize logo with the new one across all email templates
- Implemented consistent logo usage through `ME_LOGO_PNG` constant
- Updated logo fallback mechanisms for community-specific cases

2. **Email Template Migration**
- Moved email templates to Postmark for:
  - Team admin contact emails
  - Vendor submission notifications
  - Testimonial submissions
  - Action submissions
  - Event submissions
  - Admin notifications
  - User registration
  - Team/Community invitations
- Added new email template constants for better organization
- Improved error handling and logging in vendor-related operations

#### Requirements (place an `x` in each `[ ]`)
* [x] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've tested my changes manually.
* [ ] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [x] I've given my PR a meaningful title.
* [x] I linked this PR to the relevant issue.
* [ ] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [ ] I moved the linked issue to "QA Verification" now that it is ready to merge.

### CODE REVIEW

File: src/_main_/utils/constants.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
 
 STATES = load_json(BASE_DIR + "/database/raw_data/other/states.json")
 
-ME_LOGO_PNG = "https://www.massenergize.org/wp-content/uploads/2021/07/cropped-me-logo-transp.png"
+ME_LOGO_PNG = "https://www.massenergize.org/wp-content/uploads/2023/03/MassEnergize-logo.png"
```
Comment (HeyMrQuaidoo) [inline src/_main_/utils/constants.py:55]: @abdullai-t Can't we pick this from param store or .env ?

File: src/api/services/admin.py [hunk 2]
```diff
@@ -29,11 +30,16 @@ def add_super_admin(self, context, args) -> Tuple[dict, MassEnergizeAPIError]:
...
             'admintype': 'Super',
-            'admintext': "Now that you are a super admin, you have access the MassEnergize admin website at %s. You have full control over the content of our sites, can publish new communities and add new admins" % (ADMIN_URL_ROOT)
+            'admintext': "Now that you are a super admin, you have access the MassEnergize admin website at %s. You have full control over the content of our sites, can publish new communities and add new admins" % (ADMIN_URL_ROOT),
+            "me_logo": ME_LOGO_PNG,
+            "subject": "Welcome to the MassEnergize Team"
         }
+
+        send_massenergize_email_with_attachments(NEW_ADMIN_EMAIL_TEMPLATE, content_variables, [admin.email], None, None, None)
+
         # sent from MassEnergize to cadmins
```
Comment (HeyMrQuaidoo) [inline src/api/services/admin.py:40]: Let's refrain from including comments in code pushed to production. @abdullai-t

**Review by HeyMrQuaidoo — COMMENTED** [review]
Take a look at the comments.

File: src/_main_/utils/constants.py (hunk 1 above)
Comment (abdullai-t) [reply src/_main_/utils/constants.py:55]: We could push it to env

**Review by HeyMrQuaidoo — APPROVED** [review]
LGTM1

**Pull request merged**